dts/bindings/gpio/ite,it8xxx2-gpiokscan.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: This binding gives a base representation of the ITE kscan pins as gpio
+
+compatible: "ite,it8xxx2-gpiokscan"
+
+include: [base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  ngpios:
+    type: int
+    required: true
+    description: |
+      This property indicates the number of in-use slots of available slots
```

Comment:
this is a standard property found in `gpio-controller`.

Reply:
Ok, I included gpio-controller.yaml.